CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,85 @@
+# Contributor Covenant Code of Conduct
```

Comment:
We don't have `.md` files yet in lilypond.  I assume that this file should go to the documentation?  Then it should be converted to texinfo format and included...  What location do you suggest?

Reply:
My goal with this, over the next two weeks, is to modernize the top-level folder of the project and bring it in line with projects that appeal to broader communities, especially younger generations and women. In general, the expectation is that a `README.md`, `CONTRIBUTING.md` and `CODE_OF_CONDUCT.md` will be browsable at the top level of a project.

I do agree that the file should go in the documentation, and my recommendation as an initial measure would be to copy this verbatim in texinfo format in a separate PR and include it as a separate document included on the documentation page.  I think that, for this initial PR, having the code of conduct where it is will serve its primary purpose, which is to make new and infrequent contributors with a GitHub-centric workflow feel like lilypond is a welcoming and inclusive community.

What do you think?

Reply:
To avoid code duplication and to ensure synchronization it would be good to have a script that copies this to a texinfo file accordingly (while building the documentation), possibly executing some regexps for better formatting.

However, adding this file to the documentation is not urgent, so I favor inclusion in the top-level directory only, adding a TODO somewhere that such a script should be written.

On the other hand, if you think that the text is stable (for whatever definition of 'stable') I don't mind a duplication, similar to what we already have for the GPL (see files `COPYING` and `gpl.itexi`).

Reply:
Absolutely! :+1: for common sense and not creating obstacles @lemzwerg :)